# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: 0facab7b9a1d
# Full Hash: 0facab7b9a1dad599c9f5049cdf104926eace696
# Author: Kelsey Gilbert <kelsey.gilbert@mozilla.com>
# Date: 2022-12-21 21:21:23
# Regressor Bug: 1799258
# File Overlap Count: 1
# Description:
#   Bug 1799258 - Do color-management on Windows+DComp via IDCompositionFilterEffects. r=sotaro
#   
#   + Add gfx.color_management.rec709_gamma_as_srgb:true. :'(
#   
#   In particular, rec709(16/255) -> srgb(31/255). Even though it's
# ==============================================================================

diff -r 4f9ee3537468 -r 0facab7b9a1d modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Tue Dec 20 22:47:52 2022 +0000
+++ b/modules/libpref/init/all.js	Tue Dec 20 22:47:53 2022 +0000
@@ -456,8 +456,6 @@
   pref("gfx.hidpi.enabled", 2);
 #endif
 
-pref("gfx.color_management.display_profile", "");
-
 pref("gfx.downloadable_fonts.enabled", true);
 pref("gfx.downloadable_fonts.fallback_delay", 3000);
 pref("gfx.downloadable_fonts.fallback_delay_short", 100);
